.github/workflows/python-app.yml

```diff
@@ -18,12 +18,12 @@ jobs:
         sudo apt-get -y update
 
         sudo apt-get -y install python3-pil python3-yaml python3-serial python3-zmq \
-        python3-scipy python3-decorator cgroup-bin \
+        python3-scipy python3-decorator \
         python3-libtiff python3-comedilib python3-h5py libtiff5-dev python3-future \
         python3-pkg-resources python3-netifaces python3-cairo python3-numpy python3-scipy \
         python3-decorator python3-xdg python3-suds python3-matplotlib python3-opencv \
         python3-wxgtk4.0 python3-serial python3-evdev python3-notify2 python3-psutil \
-        graphicsmagick cgroup-bin authbind python3-setuptools cython python3-pyro4-delmic
```

Comment:
That one is now called "cgroup-tools". You probably want to explicitly install it. (it's need to start an odemis backend)